tests/images.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -euo pipefail
+
+export PATH="$(realpath $(dirname $0)/../target/${PROFILE:-debug}):$PATH"
+fixtures="$(realpath $(dirname $0)/../fixtures)"
+
+msg() {
+    cat <<EOF
+
+[1;34m###############################################################################[0m
+[1;34m$1[0m
+[1;34m###############################################################################[0m
+EOF
+}
+
+call() {
+    msg "$*"
+    local cmd="$1"
+    shift
+    "$(dirname $0)/images/${cmd}" "$@"
+}
+
+if [ -n "${1:-}" ]; then
```

Comment:
Hmm, kinda awkward UX. Might be simpler to just always run the fixtures-based tests and only run the cosa based ones if a base dir was given. It'll add a couple of seconds to the CoreOS CI run, but meh...